src/perf_counters.cc

```diff
@@ -254,7 +254,7 @@ bool PerfCounters::IsCounterSupported(const std::string&) { return false; }
 PerfCounters PerfCounters::Create(
     const std::vector<std::string>& counter_names) {
   if (!counter_names.empty()) {
-    GetErrorLogInstance() << "Performance counters not supported.";
+    GetErrorLogInstance() << "Performance counters not supported.\n";
```

Comment:
nit: `std::endl` is the C++ standard line ending output character.

Reply:
let me know if you want to change this or if i should just merge it.

Reply:
Just seeing this and I would just stick to `\n` to avoid a STDOUT flush, more @ https://en.cppreference.com/w/cpp/io/manip/endl.